Backport: Agents Manager 0.9.1 (#51095)
Automattic/jetpack: projects/packages/assets/CHANGELOG.md

The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).

## [4.4.10] - 2026-08-06
### Fixed

> - Script Data: decode HTML entities in the shared site title so consumers (e.g. the Newsletter Sender Settings panel) don't show literal entities like `&#039;` for site names containing an apostrophe or other special characters. [#50991]

Reviewer: Let us simplify this

```suggestion
- Script Data: Decode HTML entities in the shared site title to avoid consumers seeing the encoded entities. [#50991]
```
